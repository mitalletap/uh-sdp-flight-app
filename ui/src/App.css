.App {
  background-color: black;
  color: black;
  min-height: 100vh;
  padding: 1%;
  overflow: hidden;
  height: 100%;
}
