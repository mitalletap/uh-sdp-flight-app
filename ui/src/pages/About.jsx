import React from "react";
import { Avatar, Typography, Divider } from "antd";
import {
  GithubOutlined,
  LinkedinOutlined,
  FileTextOutlined,
  GlobalOutlined
} from "@ant-design/icons";

const { Text } = Typography;
const About = () => {
  return (
    <React.Fragment>
      <div style={{ paddingBottom: "100px" }}>
        <div
          style={{
            backgroundColor: "white",
            padding: "100px",
            height: "100%",
            width: "50%",
            textAlign: "center",
            top: "50%",
            transform: "translateX(50%) translateY(10%)"
          }}
        >
          <div className="header">
            <Text>
              <h1> MEET THE TEAM </h1>
            </Text>
          </div>
          <div className="information">
            <div
              className="Kevin"
              style={{
                display: "table",
                height: "300px",
                paddingBottom: "100px"
              }}
            >
              <div style={{ float: "left", height: "100%", width: "20%" }}>
                <Avatar
                  size={250}
                  src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
                />
              </div>
              <div
                style={{
                  float: "right",
                  textAlign: "right",
                  width: "80%",
                  height: "100%",
                  position: "relative"
                }}
              >
                <div>
                  <Text>
                    <h1 style={{ fontSize: "40px" }}> KEVIN HUYNH </h1>
                  </Text>
                  <Text>
                    {" "}
                    Kevin Huynh is a Senior at the University of Houston. He has
                    taken courses such as Data Structures, Algorithms, and
                    Database Systems.{" "}
                  </Text>
                </div>
                <div style={{ position: "absolute", bottom: "0", right: "0" }}>
                  <GithubOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                  <LinkedinOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                  <FileTextOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                  <GlobalOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                </div>
              </div>
            </div>
          </div>
          <Divider />
          <div className="information">
            <div
              className="Kevin"
              style={{
                display: "table",
                height: "300px",
                paddingBottom: "100px"
              }}
            >
              <div style={{ float: "left", height: "100%", width: "20%" }}>
                <Avatar
                  size={250}
                  src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
                />
              </div>
              <div
                style={{
                  float: "right",
                  textAlign: "right",
                  width: "80%",
                  height: "100%",
                  position: "relative"
                }}
              >
                <div>
                  <Text>
                    <h1 style={{ fontSize: "40px" }}> KEVIN HUYNH </h1>
                  </Text>
                  <Text>
                    {" "}
                    Kevin Huynh is a Senior at the University of Houston. He has
                    taken courses such as Data Structures, Algorithms, and
                    Database Systems.{" "}
                  </Text>
                </div>
                <div style={{ position: "absolute", bottom: "0", right: "0" }}>
                  <GithubOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                  <LinkedinOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                  <FileTextOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                  <GlobalOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                </div>
              </div>
            </div>
          </div>
          <Divider />
          <div className="information">
            <div
              className="Kevin"
              style={{
                display: "table",
                height: "300px",
                paddingBottom: "100px"
              }}
            >
              <div style={{ float: "left", height: "100%", width: "20%" }}>
                <Avatar
                  size={250}
                  src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
                />
              </div>
              <div
                style={{
                  float: "right",
                  textAlign: "right",
                  width: "80%",
                  height: "100%",
                  position: "relative"
                }}
              >
                <div>
                  <Text>
                    <h1 style={{ fontSize: "40px" }}> KEVIN HUYNH </h1>
                  </Text>
                  <Text>
                    {" "}
                    Kevin Huynh is a Senior at the University of Houston. He has
                    taken courses such as Data Structures, Algorithms, and
                    Database Systems.{" "}
                  </Text>
                </div>
                <div style={{ position: "absolute", bottom: "0", right: "0" }}>
                  <GithubOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                  <LinkedinOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                  <FileTextOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                  <GlobalOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                </div>
              </div>
            </div>
          </div>
          <Divider />
          <div className="information">
            <div
              className="Kevin"
              style={{
                display: "table",
                height: "300px",
                paddingBottom: "100px"
              }}
            >
              <div style={{ float: "left", height: "100%", width: "20%" }}>
                <Avatar
                  size={250}
                  src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
                />
              </div>
              <div
                style={{
                  float: "right",
                  textAlign: "right",
                  width: "80%",
                  height: "100%",
                  position: "relative"
                }}
              >
                <div>
                  <Text>
                    <h1 style={{ fontSize: "40px" }}> KEVIN HUYNH </h1>
                  </Text>
                  <Text>
                    {" "}
                    Kevin Huynh is a Senior at the University of Houston. He has
                    taken courses such as Data Structures, Algorithms, and
                    Database Systems.{" "}
                  </Text>
                </div>
                <div style={{ position: "absolute", bottom: "0", right: "0" }}>
                  <GithubOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                  <LinkedinOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                  <FileTextOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                  <GlobalOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                </div>
              </div>
            </div>
          </div>
          <Divider />
          <div className="information">
            <div
              className="Kevin"
              style={{
                display: "table",
                height: "300px",
                paddingBottom: "100px"
              }}
            >
              <div style={{ float: "left", height: "100%", width: "20%" }}>
                <Avatar
                  size={250}
                  src="https://zos.alipayobjects.com/rmsportal/ODTLcjxAfvqbxHnVXCYX.png"
                />
              </div>
              <div
                style={{
                  float: "right",
                  textAlign: "right",
                  width: "80%",
                  height: "100%",
                  position: "relative"
                }}
              >
                <div>
                  <Text>
                    <h1 style={{ fontSize: "40px" }}> KEVIN HUYNH </h1>
                  </Text>
                  <Text>
                    {" "}
                    Kevin Huynh is a Senior at the University of Houston. He has
                    taken courses such as Data Structures, Algorithms, and
                    Database Systems.{" "}
                  </Text>
                </div>
                <div style={{ position: "absolute", bottom: "0", right: "0" }}>
                  <GithubOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                  <LinkedinOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                  <FileTextOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                  <GlobalOutlined
                    style={{ fontSize: "45px", paddingRight: "10px" }}
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </React.Fragment>
  );
};

export default About;
