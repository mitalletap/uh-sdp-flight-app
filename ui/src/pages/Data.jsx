import React from "react";
import { Container, Content } from "rsuite";
import Planner from "../components/Planner";

const Data = props => (
  <React.Fragment>
    <Container>
      <Content>
        <Planner />
      </Content>
    </Container>
  </React.Fragment>
);

export default Data;
