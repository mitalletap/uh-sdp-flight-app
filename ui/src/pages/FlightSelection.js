import React from "react";
import FlightSearch from "../components/FlightSearch";
import Nav from "../components/NavigationBar";
import { Container, Header, Footer, Content, Toggle } from "rsuite";

const Home = () => {
  return (
    <React.Fragment>
      <Container>
        <Header>
          <Nav />
        </Header>
        <Content>
          <FlightSearch />
        </Content>
        <Footer></Footer>
      </Container>
    </React.Fragment>
  );
};

export default Home;
