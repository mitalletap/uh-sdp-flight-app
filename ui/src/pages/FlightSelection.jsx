import React from "react";
import { Container, Footer, Content } from "rsuite";
import FlightSearch from "../components/FlightSearch";

const FlightSelection = props => (
  <React.Fragment>
    <Container>
      <Content>
        <FlightSearch {...props} />
      </Content>
      <Footer />
    </Container>
  </React.Fragment>
);

export default FlightSelection;
