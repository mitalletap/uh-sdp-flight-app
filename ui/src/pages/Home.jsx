import React from "react";
import { Container, Content } from "rsuite";

const Home = () => {
  return (
    <React.Fragment>
      <Container>
        <Content>
          <h1> Welcome to Takeoff </h1>
        </Content>
      </Container>
    </React.Fragment>
  );
};

export default Home;
