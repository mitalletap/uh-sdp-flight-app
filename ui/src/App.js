import React from "react";
import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
import Home from "./pages/Home.jsx";
import FlightSelection from "./pages/FlightSelection.jsx";
import Data from "./pages/Data.jsx";
import Profile from "./pages/Profile.jsx";
import PageNotFound from "./pages/404";
import About from "./pages/About";
import NavBar from "./components/NavBar.jsx";
import "./App.css";
import planeImg1 from "./images/Plane1.jpg";
import planeImg2 from "./images/Plane2.jpg";
import planeImg3 from "./images/Plane3.jpg";
import planeImg4 from "./images/Plane4.jpg";
import planeImg5 from "./images/Plane5.jpg";
import { createBrowserHistory } from "history";
import Amplify from "aws-amplify";
import awsconfig from "./aws-exports";
import { withAuthenticator } from "aws-amplify-react";

// const location = history.location;
//   const unlisten = history.listen((location, action) => {
//     // location is an object like window.location
//     console.log(action, location.pathname, location.state);
//   });
// import { Auth } from "aws-amplify";
// import { CognitoUserPool } from "amazon-cognito-identity-js";
// AWS Pool Data
// const [email, setEmail] = useState("");
// const [password, setPassword] = useState("");
// const poolData = {
//   UserPoolId: "us-east-2_S2FwnCKiO",
//   ClientId: "5r6disd2l9k8kbnhu8qu6f5rn3"
// };
// const UserPool = new CognitoUserPool(poolData);

// // Modal Hook
// const [show, setShow] = useState(true);
// const currentSession = Auth.currentAuthenticatedUser()
//   .then(user => setEmail(user.attributes.email))
//   .catch(err => console.log(err));

export const history = createBrowserHistory();

const signUpConfig = {
  header: "My Customized Sign Up",
  hideAllDefaults: false,
  defaultCountryCode: "1",
  signUpFields: [
    {
      label: "Name",
      key: "name",
      required: true,
      displayOrder: 1,
      type: "string"
    }
  ]
};

Amplify.configure(awsconfig);

function App() {
  console.log(window.location.pathname);
  var backgroundImage = planeImg5;
  switch (window.location.pathname) {
    case "/":
      backgroundImage = planeImg2;
      break;
    case "/planner":
      backgroundImage = planeImg1;
      break;
    case "/about":
      backgroundImage = planeImg3;
      break;
    case "/profile":
      backgroundImage = planeImg5;
      break;
    default:
      backgroundImage = planeImg4;
  }
  return (
    <div
      className="App"
      style={{
        backgroundImage: `url(${backgroundImage})`,
        resize: "both",
        backgroundSize: "cover",
        backgroundRepeat: "no-repeat"
      }}
    >
      <NavBar />
      <Router>
        <Switch>
          <Route
            exact
            path="/"
            render={props => <FlightSelection {...props} />}
          />
          <Route exact path="/planner" render={props => <Data {...props} />} />
          <Route exact path="/logout" render={props => <Home {...props} />} />
          <Route
            exact
            path="/profile"
            render={props => <Profile {...props} />}
          />
          <Route exact path="/about" render={props => <About {...props} />} />
          <Route component={PageNotFound} />
        </Switch>
      </Router>
    </div>
  );
}

//export default App;
export default withAuthenticator(App, { signUpConfig });
