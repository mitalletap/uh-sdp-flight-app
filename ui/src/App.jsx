import React from "react";
import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
import { createBrowserHistory } from "history";
import Amplify from "aws-amplify";
import { withAuthenticator } from "aws-amplify-react";
import Home from "./pages/Home";
import FlightSelection from "./pages/FlightSelection";
import Data from "./pages/Data";
import Profile from "./pages/Profile";
import PageNotFound from "./pages/404";
import About from "./pages/About";
import NavBar from "./components/NavBar";
import "./App.css";
import planeImg1 from "./images/Plane1.jpg";
import awsconfig from "./aws-exports";

export const history = createBrowserHistory();

const signUpConfig = {
  header: "My Customized Sign Up",
  hideAllDefaults: false,
  defaultCountryCode: "1",
  signUpFields: [
    {
      label: "Name",
      key: "name",
      required: true,
      displayOrder: 1,
      type: "string"
    }
  ]
};

Amplify.configure(awsconfig);

export function App() {
  const backgroundImage = planeImg1;

  return (
    <div
      className="App"
      style={{
        backgroundImage: `url(${backgroundImage})`,
        resize: "both",
        backgroundSize: "cover",
        backgroundRepeat: "no-repeat"
      }}
    >
      <NavBar />
      <Router className="router">
        <Switch className="switch">
          <Route
            className="flightSelectionRoute"
            exact
            path="/"
            render={props => <FlightSelection {...props} />}
          />
          <Route exact path="/planner" render={props => <Data {...props} />} />
          <Route exact path="/logout" render={props => <Home {...props} />} />
          <Route
            exact
            path="/profile"
            render={props => <Profile {...props} />}
          />
          <Route exact path="/about" render={props => <About {...props} />} />
          <Route component={PageNotFound} />{" "}
        </Switch>
      </Router>
    </div>
  );
}

export default withAuthenticator(App, { signUpConfig });
