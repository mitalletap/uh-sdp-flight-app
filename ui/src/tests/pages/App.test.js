import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

import { App } from "../../App";
import Home from "../../pages/Home";
import FlightSelection from "../../pages/FlightSelection";
import Data from "../../pages/Data";
import Profile from "../../pages/Profile";
import About from "../../pages/About";

Enzyme.configure({ adapter: new Adapter() });
describe("app testing", () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<App />);
  });

  it("renders router", () => {
    expect(wrapper.find("Router")).not.toBe(null);
  });

  it("should render Flight Selection", () => {
    const route = wrapper.find("Route").first();
    const { render } = route.props();
    const props = { props: "test" };
    const rendered = render(props);
    expect(route.props().path).toEqual("/");
    expect(rendered.type).toEqual(FlightSelection);
    expect(rendered.props).toEqual(props);
  });

  it("should render Data", () => {
    const route = wrapper.find("Route").at(1);
    const { render } = route.props();
    const props = { props: "test" };
    const rendered = render(props);
    expect(route.props().path).toEqual("/planner");
    expect(rendered.type).toEqual(Data);
    expect(rendered.props).toEqual(props);
  });

  it("should render Logout", () => {
    const route = wrapper.find("Route").at(2);
    const { render } = route.props();
    const props = { props: "test" };
    const rendered = render(props);
    expect(route.props().path).toEqual("/logout");
    expect(rendered.type).toEqual(Home);
    expect(rendered.props).toEqual(props);
  });

  it("should render Profile", () => {
    const route = wrapper.find("Route").at(3);
    const { render } = route.props();
    const props = { props: "test" };
    const rendered = render(props);
    expect(route.props().path).toEqual("/profile");
    expect(rendered.type).toEqual(Profile);
    expect(rendered.props).toEqual(props);
  });

  it("should render About", () => {
    const route = wrapper.find("Route").at(4);
    const { render } = route.props();
    const props = { props: "test" };
    const rendered = render(props);
    expect(route.props().path).toEqual("/about");
    expect(rendered.type).toEqual(About);
    expect(rendered.props).toEqual(props);
  });
});
