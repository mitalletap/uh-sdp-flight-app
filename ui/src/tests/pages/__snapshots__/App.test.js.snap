// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`switch case should check snapshot 1`] = `
<div
  className="App"
  style={
    Object {
      "backgroundImage": "url(Plane1.jpg)",
      "backgroundRepeat": "no-repeat",
      "backgroundSize": "cover",
      "resize": "both",
    }
  }
>
  <div
    className="rs-container"
  >
    <div
      className="rs-content"
    >
      <img
        src="logo.png"
        style={
          Object {
            "alignItems": "center",
            "display": "block",
            "height": "50vh",
            "justifyContent": "center",
            "margin": "auto",
            "textAlign": "center",
          }
        }
      />
      <div
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
            "justifyContent": "center",
          }
        }
      >
        <div
          className="ant-select ant-select-single ant-select-show-arrow ant-select-show-search"
          onBlur={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          style={
            Object {
              "width": 400,
            }
          }
        >
          <div
            className="ant-select-selector"
            onClick={[Function]}
            onMouseDown={[Function]}
          >
            <span
              className="ant-select-selection-search"
            >
              <input
                aria-activedescendant="undefined_list_0"
                aria-autocomplete="list"
                aria-controls="undefined_list"
                aria-haspopup="listbox"
                aria-owns="undefined_list"
                autoComplete="off"
                className="ant-select-selection-search-input"
                onChange={[Function]}
                onKeyDown={[Function]}
                onMouseDown={[Function]}
                onPaste={[Function]}
                readOnly={true}
                role="combobox"
                style={
                  Object {
                    "opacity": 0,
                  }
                }
                value=""
              />
            </span>
            <span
              className="ant-select-selection-placeholder"
            >
              Depart From
            </span>
          </div>
          <span
            aria-hidden={true}
            className="ant-select-arrow"
            onMouseDown={[Function]}
            style={
              Object {
                "WebkitUserSelect": "none",
                "userSelect": "none",
              }
            }
            unselectable="on"
          >
            <span
              aria-label="down"
              className="anticon anticon-down"
              role="img"
            >
              <svg
                aria-hidden="true"
                className=""
                data-icon="down"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                />
              </svg>
            </span>
          </span>
        </div>
        <div
          className="ant-select ant-select-single ant-select-show-arrow ant-select-show-search"
          onBlur={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          style={
            Object {
              "width": 400,
            }
          }
        >
          <div
            className="ant-select-selector"
            onClick={[Function]}
            onMouseDown={[Function]}
          >
            <span
              className="ant-select-selection-search"
            >
              <input
                aria-activedescendant="undefined_list_0"
                aria-autocomplete="list"
                aria-controls="undefined_list"
                aria-haspopup="listbox"
                aria-owns="undefined_list"
                autoComplete="off"
                className="ant-select-selection-search-input"
                onChange={[Function]}
                onKeyDown={[Function]}
                onMouseDown={[Function]}
                onPaste={[Function]}
                readOnly={true}
                role="combobox"
                style={
                  Object {
                    "opacity": 0,
                  }
                }
                value=""
              />
            </span>
            <span
              className="ant-select-selection-placeholder"
            >
              Arrive In
            </span>
          </div>
          <span
            aria-hidden={true}
            className="ant-select-arrow"
            onMouseDown={[Function]}
            style={
              Object {
                "WebkitUserSelect": "none",
                "userSelect": "none",
              }
            }
            unselectable="on"
          >
            <span
              aria-label="down"
              className="anticon anticon-down"
              role="img"
            >
              <svg
                aria-hidden="true"
                className=""
                data-icon="down"
                fill="currentColor"
                focusable="false"
                height="1em"
                viewBox="64 64 896 896"
                width="1em"
              >
                <path
                  d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                />
              </svg>
            </span>
          </span>
        </div>
      </div>
      <div
        style={
          Object {
            "alignItems": "center",
            "display": "flex",
            "height": "10vh",
            "justifyContent": "center",
          }
        }
      >
        <div
          style={
            Object {
              "paddingRight": "10px",
            }
          }
        >
          <div
            className="ant-picker"
            onMouseUp={[Function]}
          >
            <div
              className="ant-picker-input"
            >
              <input
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onMouseDown={[Function]}
                placeholder="Start"
                readOnly={true}
                size={12}
                title=""
                value=""
              />
              <span
                className="ant-picker-suffix"
              >
                <span
                  aria-label="calendar"
                  className="anticon anticon-calendar"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    className=""
                    data-icon="calendar"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M880 184H712v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H384v-64c0-4.4-3.6-8-8-8h-56c-4.4 0-8 3.6-8 8v64H144c-17.7 0-32 14.3-32 32v664c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V216c0-17.7-14.3-32-32-32zm-40 656H184V460h656v380zM184 392V256h128v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h256v48c0 4.4 3.6 8 8 8h56c4.4 0 8-3.6 8-8v-48h128v136H184z"
                    />
                  </svg>
                </span>
              </span>
            </div>
          </div>
        </div>
        <div
          style={
            Object {
              "paddingRight": "10px",
              "width": 100,
            }
          }
        >
          <div
            className="ant-input-number"
            style={Object {}}
          >
            <div
              className="ant-input-number-handler-wrap"
            >
              <span
                aria-disabled={false}
                aria-label="Increase Value"
                className="ant-input-number-handler ant-input-number-handler-up "
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchCancel={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="button"
                unselectable="unselectable"
              >
                <span
                  aria-label="up"
                  className="anticon anticon-up ant-input-number-handler-up-inner"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    className=""
                    data-icon="up"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M890.5 755.3L537.9 269.2c-12.8-17.6-39-17.6-51.7 0L133.5 755.3A8 8 0 00140 768h75c5.1 0 9.9-2.5 12.9-6.6L512 369.8l284.1 391.6c3 4.1 7.8 6.6 12.9 6.6h75c6.5 0 10.3-7.4 6.5-12.7z"
                    />
                  </svg>
                </span>
              </span>
              <span
                aria-disabled={true}
                aria-label="Decrease Value"
                className="ant-input-number-handler ant-input-number-handler-down ant-input-number-handler-down-disabled"
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                role="button"
                unselectable="unselectable"
              >
                <span
                  aria-label="down"
                  className="anticon anticon-down ant-input-number-handler-down-inner"
                  role="img"
                >
                  <svg
                    aria-hidden="true"
                    className=""
                    data-icon="down"
                    fill="currentColor"
                    focusable="false"
                    height="1em"
                    viewBox="64 64 896 896"
                    width="1em"
                  >
                    <path
                      d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                    />
                  </svg>
                </span>
              </span>
            </div>
            <div
              className="ant-input-number-input-wrap"
            >
              <input
                aria-valuemax={10}
                aria-valuemin={1}
                aria-valuenow={1}
                autoComplete="off"
                className="ant-input-number-input"
                max={10}
                min={1}
                onBlur={[Function]}
                onChange={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseUp={[Function]}
                required={false}
                role="spinbutton"
                step={1}
                value="1"
              />
            </div>
          </div>
        </div>
        <div
          style={
            Object {
              "paddingRight": "10px",
            }
          }
        >
          <button
            aria-checked={false}
            className="ant-switch"
            defaultChecked={false}
            onChange={[Function]}
            onClick={[Function]}
            onKeyDown={[Function]}
            onMouseUp={[Function]}
            role="switch"
            size="default"
            type="button"
          >
            <span
              className="ant-switch-inner"
            >
              OW
            </span>
          </button>
        </div>
        <div
          style={
            Object {
              "paddingRight": "10px",
            }
          }
        >
          <button
            className="ant-btn ant-btn-primary"
            onClick={[Function]}
            onMouseDown={[Function]}
            style={
              Object {
                "marginRight": "10px",
              }
            }
            type="button"
          >
            <span>
              Find Flights
            </span>
          </button>
        </div>
      </div>
    </div>
    <div
      className="rs-footer"
    />
  </div>
</div>
`;
